helloworld/README.md

```diff
@@ -1,23 +1,199 @@
-# Helloworld for Dubbo-go 3.0
-
-For api definition and go client and server startup, please refer to [dubbo-go 3.0 quickstart](https://dubbogo.github.io/zh-cn/docs/user/quickstart/3.0/quickstart.html)
-
-## Instructions
-1. Start zookeeper
-
-2. Start the server
-
-Use goland to start helloworld-go-server
-
-or
-
-Execute `sh run.sh` in the java-server folder to start the java server
-
-3. Start the client
-
-Use goland to start helloworld-go-client
-
-or
-
-Execute `sh run.sh` under the java-client folder to start the java client
-
+# Helloworld for dubbo-go Triple
+
+This is **Triple** helloworld example to help you finish a basic RPC invocation done quickly.
+
+## Prerequisites
+
+### install protoc
+
+#### install with package manager
+
+```shell
+# for Linux, use apt or apt-get
+apt install -y protobuf-compiler
+protoc --version # make sure that version is 3+
+
+# for Macos, using homebrew
+brew install protobuf
+protoc --version # make sure that version is 3+
+```
+
+#### install pre-compiled binary
+
+```shell
+# fetch the pre-compiled protoc corresponding to your operating system and computer architecture
+# protoc-<version>-<os>-<arch>.zip
+# or you can download from github.com/protocolbuffers/protobuf/releases manually
+PB_REL="https://github.com/protocolbuffers/protobuf/releases"
+curl -LO $PB_REL/download/v24.4/protoc-24.4-linux-x86_64.zip
+
+# unzip downloaded file under a directory
+unzip protoc-24.4-linux-x86_64.zip -d $HOME/.local
+
+# update path variable
+export PATH="$PATH:$HOME/.local/bin"
+```
+
+#### build protoc from sources
+
+Please see [**Download Protocol Buffers**](https://protobuf.dev/downloads/).
+
+### install protoc-gen-go
+
+```shell
+# install the version of your choice of protoc-gen-go. here use the latest version as example
+go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31
+```
+
+### install protoc-gen-triple
```

Comment:
Should we rename to `protoc-gen-triple-go` or `protoc-gen-go-triple` to follow the protoc plugin naming convention?